modules/nci/include/imanager.h

```diff
@@ -48,7 +49,7 @@ class IManager
     void add(F frame, Point2D origin = Point2D(0, 0), Size2D parent_size = Size2D(0, 0))
     {
         parent_size = get_size();
-        _children.add(frame, origin, parent_size);
+        nci::IManager::_children.add(frame, origin, parent_size);
```

Comment:
nci namespace is really needed here? Remove if compile without.

Reply:
It works without nci namespace.